OREAnalytics/orea/app/reportwriter.cpp

```diff
@@ -166,10 +111,7 @@ void ReportWriter::writeNpv(ore::data::Report& report, const std::string& baseCu
                 .add(nullString_)
                 .add(Null<Real>())
                 .add(nullString_)
-                .add(nullString_)
-                .add(Null<Real>())
-                .add(Null<Real>())
-                .add(Null<Real>());
```

Comment:
why 3 components removed here when all the others were 4 reals?

Reply:
That's the bug I accidentally created, it should've been four and the missing one caused the "only 16 entries filled" error hehe 🤦‍♂️